src/_community/rules.md

```diff
@@ -46,14 +46,26 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted **in all Quilt community spaces** must meet the following criteria:
```

Comment:
I know that someone said to emphasize this bit, and I agree with that, but perhaps say "in all ***official*** Quilt community spaces" to make it clear that some unofficial Quilt community space wouldn't be subject to these rules. *I know that it's pretty obvious but some people, namely me, are dense and might not get that.*

Reply:
Well, there are two things here.

1. It doesn't really make sense for people to assume these rules will be applied outside the places the community team watches over
2. Unofficial communities run by Quilt team members (like the IRC channel) and potentially other Quilt-related communities are welcome to make use of the same policies, so specifying "official" here would likely result in the same rules-lawyering you're implying, haha

For those reasons, I don't think this is a pointful change